.github/workflows/release.yml

```diff
@@ -21,38 +21,46 @@ jobs:
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           repository: wingtk/gvsbuild
           fetch-depth: 0
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python }}
           architecture: ${{ matrix.arch }}
 
       - name: Install Build Deps
         run: python -m pip install wheel
 
```

Comment:
Okay after further investigation we don't need to install wheel since this is done as part of the gvsbuild process and can be removed

https://github.com/wingtk/gvsbuild/blob/49fde77d095cfd12c5f00ede32be8b7190398bd5/gvsbuild/tools.py#L298

Reply:
I thought that only happened if python was not already setup.

Reply:
Okay, I maybe understood, ignore me then on this one  :smile: